tests/lib/test_utils.py

```diff
@@ -144,6 +145,14 @@ def test_delete_from_queue_bad_message(sqs, queue):
         utils.delete_from_queue(msg)
 
 
+@pytest.mark.parametrize("item", item_dir.glob("*.json"))
```

Comment:
Maybe this testing is overkill; I tried to follow existing test patterns in this repository, but I'm definitely open to alternative suggestions.